### ISSUE TITLE
Make this a template repository

### ISSUE THREAD

**Issue by fwcd** [svenstaro/rust-web-boilerplate#55]
![image](https://user-images.githubusercontent.com/30873659/164944729-767743f2-74d8-4989-8e16-43c158847afd.png)

**Comment by svenstaro** [comment]
Good idea. I want to revamp this repo with a much more modern version anyway and then I might as well make it a template repository. Currently this is quite old and should probably not be used for modern projects.

**Comment by urbanisierung** [comment]
Hi @svenstaro! Thanks for this repo! I'd like to get started with Rust this year and want to implement a basic http server as a goal (2-3 routes, logging, auth). Since you don't recommend this repo as starter, do you have any plans when you want to update this repo or are you aware of any other rust web boilerplate repos?

Many thanks in advance!
